test/test_brownian.py

```diff
@@ -247,7 +256,74 @@ def test_conditional_statistics(levy_area, use_levy):
             h_std = math.sqrt(h_var)
             normalised_hh = (h_r - h_mean) / h_std
             _, pval_h = stats.kstest(normalised_hh, stats.norm.cdf)
-            assert pval_h > 0.1 / (len(ts) - 2)
+            pvals_h.append(pval_h)
+    return jnp.array(pvals_w1), jnp.array(pvals_w2), jnp.array(pvals_h)
+
+
+@pytest.mark.parametrize("levy_area", ["", "space-time"])
+@pytest.mark.parametrize("use_levy", (False, True))
+def test_conditional_statistics(levy_area: LevyArea, use_levy):
+    pvals_w1, pvals_w2, pvals_h = conditional_statistics(
+        levy_area, use_levy, tol=2**-8, spacing=2**-10, spline="sqrt", min_num_points=90
+    )
+    assert jnp.all(pvals_w1 > 0.1 / pvals_w1.shape[0])
+    if levy_area == "space-time" and use_levy:
+        assert jnp.all(pvals_w2 > 0.1 / pvals_w2.shape[0])
+        assert jnp.all(pvals_h > 0.1 / pvals_h.shape[0])
+    else:
+        assert len(pvals_w2) == 0
+        assert len(pvals_h) == 0
+
+
+@pytest.mark.parametrize("levy_area", ["", "space-time"])
+@pytest.mark.parametrize("use_levy", (False, True))
+@pytest.mark.parametrize("spline", ("quad", "sqrt", "zero"))
+def test_spline(levy_area: LevyArea, use_levy, spline):
+    if levy_area == "space-time" and spline == "quad":
+        pytest.skip("Quad spline is not implemented for space-time Levy area")
```

Comment:
FYI, a skipped test is usually used to refer to something that is currently broken, but for which we just don't to worry about right now. In particular it still shows up in the output test log each time.
This means that skips should usually only be used in the short-term, not in the long-term.

Anyway, here you probably want this:
```python
def _levy_area_spline():
    for levy_area in ("", "space-time"):
        for spline in ("quad", "sqrt", "zero"):
            if levy_area == "space-time" and spline == "quad":
                continue
            yield levy_area, spline

@pytest.mark.parametrize("levy_area,spline", _levy_area_spline())
```
which won't even generate the test to be skipped.

Reply:
Thanks, that's good to know.